package.json

```diff
@@ -4,13 +4,8 @@
     "start": "node index.js"
   },
   "dependencies": {
-      "code": "^5.2.4",
-     "discord.js": "^12.3.1",
-    "fs": "0.0.1-security",
-     "path": "^0.12.7",
-     "quick.db": "^7.1.2"
-   },
-  "engines": {
-    "node": "12.x"
+    "@hapi/code": "^8.0.3",
+    "discord.js": "^12.5.1",
+    "quick.db": "^7.1.3"
```

Comment:
We require `path` and `fs` packages here as well.

Reply:
i tested and i think this packages are bundeled with node becuse i deleted node_modules and installed required packsges woth --save option

Reply:
Makes sense, Alright then.